Sources/App/bootstrap.swift

```diff
@@ -0,0 +1,21 @@
+import Service
+import Vapor
+import Foundation
+
+public func bootstrap() throws -> Application {
```

Comment:
A comment here explaining what this method does and where it's called from would probably be very appreciated!